src/force-app/main/default/lwc/createNewCase/createNewCase.js

```diff
@@ -0,0 +1,30 @@
+import { LightningElement, api, track, wire  } from 'lwc';
+import { NavigationMixin } from "lightning/navigation";
+import getNewCaseParameters from '@salesforce/apex/CreateCaseController.getNewCaseParameters';
```

Comment:
This could be done w/o Apex controller at all. 
See links below:

- https://salesforce.stackexchange.com/questions/172096/getting-sobject-type-into-lightning-component;
- https://developer.salesforce.com/docs/platform/lwc/guide/reference-wire-adapters-record.html.

Howerver, no time added.